setup.py

```diff
@@ -19,6 +19,7 @@
     'qrcode',
     'pillow',
     'cryptography',
+    'typing',
```

Comment:
ik hoop dat dit goed gaat met RPMs draaien op oude platforms, maar dat vinden we later uit.